Makefile

```diff
@@ -50,12 +52,16 @@ export K_OPTS
 
 .PHONY: all clean distclean install uninstall                                         \
         deps k-deps libsecp256k1 libff plugin-deps hook-deps                          \
-        build build-avm build-kavm                                                    \
-        test test-avm                                                                 \
-        venv venv-clean kavm-algod
+        build build-avm build-kavm py-kavm                                            \
+        test test-avm-semantics test-avm-semantics-prove                              \
+	test-kavm test-kavm-kast test-kavm-kast-avm-scenario test-kavm-kast-teal      \
+	clean-avm clean-kavm                                                          \
+	module-imports-graph module-imports-graph-dot                                 \
+        venv venv-clean
```

Comment:
Make sure your formatting the same as the surrounding code, and that you do self-review in GitHub before asking for review, to catch these space/tab issues. Use spaces here.

Reply:
Sorry about that!